### DISCUSSION TITLE
Misunderstanding of mfem::mesh vertices format

### DISCUSSION THREAD

**Discussion by moroz3339** [mfem/mfem#3210] [Q&A] [answered]
Hello everyone!
I am trying to solve the linear elasticity problem by mfem. I have some misunderstanding what is correct values of the mesh's vertex. Must It be normalized in the range [-1,1] and further scale calculated displacement appropriate real model or can I pass vertices appropriate to real model sizes into mfem::mesh constructor directly?

**Comment by tzanio** [comment, marked as answer]
Hi @moroz3339,

The vertices do not need to be normalized, you can pass the real coordinates.

Tzanio

**Comment by moroz3339** [comment]
Hi @tzanio, thank you for your answer
